controllers/calendar.js

```diff
@@ -17,10 +17,13 @@ router.get('/', (req, res, next) => {
 		res.locals.currentSchoolData.features || []
 	).includes('videoconference');
 	const showVideoconferenceOption = schoolUsesVideoconferencing;
+	const timezone = timesHelper.getChangedTimezoneString();
+
 	res.render('calendar/calendar', {
 		title: res.$t('calendar.headline.calendar'),
 		userId: res.locals.currentUser._id,
 		showVideoconferenceOption,
+		timezone,
```

Comment:
We should use global variables here: https://ddcode.net/2019/04/20/how-does-express-handlebars-pass-global-variables-to-templates/